sxt/multiexp/pippenger2/multiexponentiation.h

```diff
@@ -169,18 +172,8 @@ xena::future<> multiexponentiate_impl(basct::span<T> res,
       });
 
   // complete the multi-exponentiation by splitting the remaining work by output
```

Comment:
Does this comment still make sense here?

Reply:
Updated. Thank you.